src/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -348,7 +348,7 @@ private ZipArchiveEntry DoCreateEntry(String entryName, CompressionLevel? compre
         {
             Contract.Ensures(Contract.Result<ZipArchiveEntry>() != null);
 
-            // Checking of compressionLevel is passed down to DeflateStream and the IDeflater implementation
+            // Checking of compressionLevel is passed down to DeflateStream and the Deflater implementation
```

Comment:
Nit: This comment seems out-of-date.  Deflator is no longer "plugable" (nor is it "pugable" ;)  This same comment appears in a few other places.